sdk/typescript/test/e2e/dwallet-mpc.test.ts

```diff
@@ -133,7 +140,10 @@ describe('Test dWallet MPC', () => {
 	});
 
 	it('should sign full flow with on-chain network DKG output', async () => {
-		const networkDecryptionKeyPublicOutput = await getNetworkDecryptionKeyPublicOutput(conf, null);
+		const networkDecryptionKeyPublicOutput = await getNetworkDecryptionKeyPublicOutputID(
```

Comment:
fix the error here 
![image](https://github.com/user-attachments/assets/2f00f6fd-f17d-45e4-ac4f-1e5d893db828)
See same error in `sign()`